openmoc/process.py

```diff
@@ -213,6 +215,48 @@ def compute_fission_rates(solver, use_hdf5=False):
         pickle.dump(fission_rates_sum, open(directory + filename + '.pkl', 'wb'))
 
 
+def get_sigma_by_group(material, rxn_type, g):
+    """Return the cross section for a given reaction
+
+    Parameters
+    ----------
+    material : openmoc.Material
+        Material that we are getting the cross sections from
+    rxn_type : {'flux', 'total', 'absorption', 'scatter',
+                'fission', 'nu-fission'}
+        Reaction type we are loading the cross section for
+    g : integer
+        Energy group index (starts at 0).
+
+    Returns
+    -------
+    sigma : float
+        The cross section for energy group `g' of reaction `rxn_type'.
+        For scatter, this includes self-scatter and outscatter.
+
+    """
+    global rxn_types
+    cv.check_value('rxn_type', rxn_type, rxn_types)
+    
+    # Energy groups start at 1 in OpenMOC
+    if rxn_type == "total":
+        return material.getSigmaTByGroup(g + 1)
+    elif rxn_type == "absorption":
```

Comment:
remove absorption unless you add a "getSigmaAByGroup"

Reply:
done